README.md

```diff
@@ -98,31 +98,31 @@ yarn build
 yarn ts
 ```
 
-2. Create a new branch, and make sure to push it. This is where package version updates, and git tags will be pushed from.
+2. Create a new branch. This is where package version updates will be pushed fromm.
```

Comment:
Typo: `fromm` => `from`

Reply:
Even with my VSCode package that's supposed to safeguard me from these mistakes!